src/value.rs

```diff
@@ -580,7 +601,7 @@ macro_rules! box_to_opt_ref {
 #[cfg(feature = "with-json")]
 impl Value {
     pub fn is_json(&self) -> bool {
-        return matches!(self, Self::Json(_));
```

Comment:
We could refactor these `return`s on master branch

Reply:
@billy1624 You mean split PR?

Reply:
Yes

Reply:
@billy1624 I make new PR: https://github.com/SeaQL/sea-query/pull/310. After merge I will rebase this PR.

Reply:
#310 was merged :)